### DISCUSSION TITLE
how to use Chinese Vectorizers in BERTopic

### DISCUSSION THREAD

**Discussion by chonchonj23** [MaartenGr/BERTopic#1499] [Q&A]
Dear Maarten,

If there is a need to analyze some Chinese texts, can I use jieba as the vectorizer in BERTopic?

The syntax of jieba looks something like below:

import jieba
text = "今天天气真好"
words = jieba.cut(text)
for token in words:
  print(token)

Since it uses .cut() instead of .fit_transform(), I don't know how to load it into BERTopic.

Thank you very much for your time and help.

Best regards
Chon

**Comment by MaartenGr** [comment]
You can find the answer in the [FAQ](https://maartengr.github.io/BERTopic/faq.html#how-can-i-use-bertopic-with-chinese-documents). Hope it helps!
